erpnext/templates/pages/order.py

```diff
@@ -79,3 +80,15 @@ def get_attachments(dt, dn):
 	return frappe.get_all("File",
 		fields=["name", "file_name", "file_url", "is_private"],
 		filters={"attached_to_name": dn, "attached_to_doctype": dt, "is_private": 0})
+
+@frappe.whitelist()
+def duplicate_purchase_order(quotation_name):
```

Comment:
It's better to have this function in `cart.py`

Reply:
And also have the option to select a different DocType. And it also needs server-side validation so that the user cannot access other's Quotations or a cancelled Quoatation